app/app/urls.py

```diff
@@ -610,8 +610,9 @@
     ]
 
 urlpatterns += [
-    re_path(r'^(.*)/(.*)?', dashboard.views.profile, name='profile_min_by_tab'),
-    re_path(r'^(.*)', dashboard.views.profile, name='profile_min'),
+    re_path(r'^([a-z|A-Z|0-9|\.]+)/([a-z|A-Z|0-9|\.]+)/?$', dashboard.views.profile, name='profile_min'),
```

Comment:
the name should be `profile_min_by_tab` here right?

Reply:
tested in my local working fine with /username /username/area profile/username and profile/username/area

Reply:
@danlipert yes

note to self : figure out what non alphanumeric chars we want to support so that usernames like http://localhost:8000/virtual-face work.. maybe just anything thats not a forward slash?

Reply:
https://github.com/shinnn/github-username-regex `/^[a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}$/i`

Reply:
true slash profiles will fail here

Reply:
Have a fix commiting here

Reply:
https://regex101.com/r/wSi6ou/1